---
string next_step
bool if_measure_after_movement
geometry_msgs/PoseStamped goal_pose